forge-gui/res/cardsfolder/upcoming/tasha_unholy_archmage.txt

```diff
@@ -0,0 +1,13 @@
+Name:Tasha, Unholy Archmage
+ManaCost:2 U B
+Types:Legendary Planeswalker Tasha
+Loyalty:4
+A:AB$ Effect | Cost$ AddCounter<1/LOYALTY> | Planeswalker$ True | Triggers$ AttackTrigger | Duration$ UntilYourNextTurn | AILogic$ Main2 | SpellDescription$ Until your next turn, whenever a creature attacks you or CARDNAME, put a -1/-1 counter on that creature.
+SVar:AttackTrigger:Mode$ Attacks | ValidCard$ Creature | Attacked$ You,Card.EffectSource | TriggerZones$ Command | Execute$ TrigPutCounter | TriggerDescription$ Whenever a creature attacks you or EFFECTSOURCE, put a -1/-1 counter on that creature.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ TriggeredAttackerLKICopy | CounterType$ M1M1 | CounterNum$ 1
+A:AB$ ChooseCard | Cost$ SubCounter<2/LOYALTY> | Planeswalker$ True | ValidTgts$ Opponent | TgtPrompt$ Select target opponent | Choices$ Creature.TargetedPlayerOwn | ChoiceZone$ Graveyard | AILogic$ WorstCard | Amount$ 1 | SubAbility$ DBChangeZone | Mandatory$ True | StackDescription$ {p:Targeted} puts a creature card of their choice from their graveyard onto the battlefield under your control. That creature gains ward 2. | SpellDescription$ Target opponent puts a creature card of their choice from their graveyard onto the battlefield under your control. That creature gains ward {2}.
+SVar:DBChangeZone:DB$ ChangeZone | Defined$ ChosenCard | Origin$ Graveyard | GainControl$ True | Destination$ Battlefield | StackDescription$ None | SubAbility$ DBPump
+SVar:DBPump:DB$ Animate | Keywords$ Ward:2 | Defined$ ChosenCard | Duration$ Permanent | StackDescription$ None | SubAbility$ DBCleanup
```

Comment:
`StackDescription$ That creature gains ward 2.` is fine here IMO

Reply:
Kept in Ward 2 StackDesc in Animate ability.